drivers/arm/i2c_master.h

```diff
@@ -73,19 +73,19 @@
 // The default timing values below configures the I2C clock to 400khz assuming a 72Mhz clock
```

Comment:
This comment needs fixing

Reply:
No this is correct:)
The setting were giving 150khz we wanted 400

Reply:
My only concern with this change is it may break splits. We've had issues in the past with splits running at 400KHz failing because of excessively long trrs cables.